app/controllers/dashboards_controller.rb

```diff
@@ -5,6 +5,8 @@ class DashboardsController < ApplicationController
   before_action :redirect_to_settings_strong_mfa_required, if: :mfa_required_weak_level_enabled?
 
   def show
+    add_breadcrumb t("breadcrumbs.dashboard"), dashboard_path
```

Comment:
Having the breadcrumb in the action method kinda feels like it's blurring the intention. Could we take inspiration from https://github.com/weppos/breadcrumbs_on_rails and have `add_breadcrumb` as a class method?